buddy-guide.md

```diff
@@ -0,0 +1,67 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Buddy Guide](#buddy-guide)
+- [Before the new student starts](#before-the-new-student-starts)
+- [Day 1](#day-1)
+- [Week 1](#week-1)
+- [End of Week 1](#end-of-week-1)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Buddy Guide
+
+As part of the onboarding process, each new student will be allocated a buddy. A buddy is typically an experienced team member who will be
+able to provide training, instruction and leadership for the new student in the crucial first number of weeks. As such, we need to provide
+a guide for what we expect of you as a buddy to help the initial days of the new students setup. This guide will not cover HOW to train
+someone, we will leave that in your expert hands, it is designed to ensure the smooth and fast ramp up of the new student.
+
+
+# Before the new student starts
+
+* Ensure you are familiar with who your new student will be. Read their CV, learn something about them
+* Ensure you have the new students account details ready
+* Ensure there is space and hardware available for them
+* Create a permanent Bluejeans room that you can share with your buddy for the duration of your training together
+* Reach out to specialists to schedule complementary training for Week 1
+
+# Day 1
+
```

Comment:
Maybe another point to add here is to ensure that the  student have the equipment they need (i.e. laptop, access badge?, etc. )